### PR TITLE
[JR] lambdatest config for 2024.1.1

### PR DESCRIPTION
**Pull request by jamesreadhead-te**

### CODE REVIEW

File: Ruby/template/default_config/browser.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 &defaults_browser
 
-source: local # local, browserstack, perfecto, sauce_labs, kobiton, local_device
+source: local               # local, browserstack, perfecto, sauce_labs, kobiton, local_device
```
Comment (marksweetlove-te) [inline Ruby/template/default_config/browser.yml]: you didn't add lambda_test to the commented examples, if that's something we still want to have

**Review by marksweetlove-te — CHANGES_REQUESTED** [review]

File: Ruby/template/default_config/browser.yml (hunk 1 above)
Comment (jamesreadhead-te) [reply Ruby/template/default_config/browser.yml]: done

File: Ruby/template/default_config/browser.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 &defaults_browser
 
-source: local               # local, browserstack, perfecto, sauce_labs, kobiton, local_device
+source: local               # local, browserstack, perfecto, sauce_labs, kobiton, local_device, lambda_test - review comments
```
Comment (marksweetlove-te) [inline Ruby/template/default_config/browser.yml]: just lambda_test will do :D

**Review by marksweetlove-te — CHANGES_REQUESTED** [review]

File: Ruby/template/default_config/browser.yml (hunk 2 above)
Comment (jamesreadhead-te) [reply Ruby/template/default_config/browser.yml]: omg....who knows.... done...

**Review by marksweetlove-te — APPROVED** [review]

**Pull request merged**